retag-image

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env perl
+
+use File::Temp qw/ tempfile /;
+use File::Which;
+
+my $default_tag='latest';
+
+sub usage {
+    print STDERR "$0 IMAGE_TO_RETAG [TAG]
+
+Use manifest-tool to assign an additional TAG to IMAGE_TO_RETAG.
+
+    TAG defaults to ${default_tag}
+";
+}
+
+unless (which('manifest-tool')) {
+    print STDERR "Please install manifest-tool <https://github.com/estesp/manifest-tool>";
```

Comment:
@jsoref  I noticed you removed the branch but I wanted to update you on what happened to my progress.   I did remove the redirection and I then got passed that but then got the infamous qemu error that a lot of folks are getting with arm.   
'#0 64.71 [STARTED] Task without title.
#0 73.72 [FAILED] Cypress failed to start.
#0 73.72 [FAILED] 
#0 73.72 [FAILED] This may be due to a missing library or dependency. https://on.cypress.io/required-dependencies
#0 73.72 [FAILED] 
#0 73.72 [FAILED] Please refer to the error below for more details.
#0 73.72 [FAILED] 
#0 73.72 [FAILED] ----------
#0 73.72 [FAILED] 
#0 73.72 [FAILED] qemu: uncaught target signal 5 (Trace/breakpoint trap) - core dumped
#0 73.72 [FAILED] qemu: uncaught target signal 5 (Trace/breakpoint trap) - core dumped
#0 73.72 [FAILED] qemu: uncaught target signal 11 (Segmentation fault) - core dumped
#0 73.72 [FAILED] 
#0 73.72 [FAILED] ----------
#0 73.72 [FAILED] 
#0 73.72 [FAILED] Platform: linux-x64 (Debian - 10.11)
#0 73.72 [FAILED] Cypress Version: 10.3.0
#0 73.73 Cypress failed to start.
#0 73.73 
#0 73.73 This may be due to a missing library or dependency. https://on.cypress.io/required-dependencies
#0 73.73 
#0 73.73 Please refer to the error below for more details.
#0 73.73 
#0 73.73 ----------
#0 73.73 
#0 73.73 qemu: uncaught target signal 5 (Trace/breakpoint trap) - core dumped
#0 73.73 qemu: uncaught target signal 5 (Trace/breakpoint trap) - core dumped
#0 73.73 qemu: uncaught target signal 11 (Segmentation fault) - core dumped'

Reply:
You can still access them, either by giving a new branch name to a thing, by visiting https://github.com/cypress-io/cypress-docker-images/commits/b3465b26da55d3c41febc56dab3f5364ae9dd78e and using the branch chooser.

or by doing something like `git fetch origin refs/pull/696/head:my-new-branch`

As for the error, the goal is to run the arm version of cypress on our M1s, not the x64 versions.

Reply:
@rnunezil: if you point me to the specific shas you're using and commands you're running, I'm happy to try to help.

(This isn't my primary task, but we do use cypress, so I'm glad to try to help out.)